apps/aecore/src/aec_sync.erl

```diff
@@ -330,6 +330,7 @@ get_next_work_item(ST = #sync_task{ pool = [{_, _, {_, _}} | _] = Pool, adding =
         _ when length(Pend) < 10 orelse NewPool /= [] ->
             get_next_work_item(ST#sync_task{ pool = NewPool, pending = Pend ++ [ToBeAdded] });
         _ ->
+            epoch_sync:info("Nothing to do", []),
```

Comment:
Wait, where did this come from....

Reply:
No, I don't like this one. But I can perhaps be convinced.

Reply:
Removed.

Reply:
Context was https://github.com/aeternity/aeternity/pull/2159#issuecomment-463761595